administrator/language/en-GB/com_joomlaupdate.ini

```
@@ -15,26 +16,45 @@ COM_JOOMLAUPDATE_CONFIG_UPDATESOURCE_NEXT="Joomla Next"
 COM_JOOMLAUPDATE_CONFIG_UPDATESOURCE_TESTING="Testing"
 COM_JOOMLAUPDATE_CONFIGURATION="Joomla Update: Options"
 COM_JOOMLAUPDATE_CONFIRM="Confirm"
+COM_JOOMLAUPDATE_EMPTYSTATE_APPEND="Update your site by uploading the update package manually."
+COM_JOOMLAUPDATE_EMPTYSTATE_BUTTON_ADD="Check for updates"
+COM_JOOMLAUPDATE_EMPTYSTATE_CONTENT="Click the button below to check for updates."
```

```suggestion
COM_JOOMLAUPDATE_EMPTYSTATE_CONTENT="Select the button below to check for updates."
```

you can't click with a finger or screen reader ;)